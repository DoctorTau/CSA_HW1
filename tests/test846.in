506
696 4332 -9762 -9348 8301 -9947 850 1840 -1114 3635 -629 -888 -5240 -6238 6675 3672 -839 -51 -5513 8528 -6091 -2241 6442 -5831 7155 -6919 -6505 -363 -5448 49 1145 773 31 -5524 2558 4060 8627 -5395 -3314 -7094 5112 7715 -3514 -8204 6486 -3180 1090 6532 8960 -767 8595 -9923 -8651 -6566 -9516 -2050 7549 7848 2804 -1742 4347 5854 9223 -630 7280 -2426 493 -1752 8573 2217 8800 -1400 -2660 404 -9420 -1029 -1125 9614 -4633 -6549 -4800 -7734 5734 8660 -1051 -1967 5327 9707 3540 -8398 4866 -6417 7032 123 4796 9531 6440 -5988 -3583 -3227 -9250 4314 324 -7286 8628 1604 -1296 1475 -8109 -6154 2168 9475 8078 -6803 -4667 5003 8111 -9335 -9382 -2005 9179 9557 8124 5839 966 799 -2240 -2858 -6979 -9935 7410 -2865 7628 1905 227 674 -2061 -4544 2079 4178 3298 1956 9591 41 4105 -7217 -5437 -5599 2860 140 -9255 -9531 -3469 3453 -8237 7965 -7552 1686 -5882 -5531 -2908 -3604 -2521 -5610 -6671 7455 -1349 7712 -3085 5630 -559 -258 7658 9466 7234 5638 1072 -6370 -2184 -8492 -1504 3099 -1265 -5722 -8947 -4231 7895 7255 -5564 2203 -7622 -6657 -8019 -8646 -7551 7813 357 4989 -6989 -9864 -2351 9009 -8601 7692 -3895 1253 -202 -1520 -9058 -7087 -2966 -4982 -4753 2866 -4318 6844 5233 1176 -170 8108 4474 7484 -4964 -2766 -4392 -9539 7098 -5254 1724 4056 3840 182 4231 -7204 6424 907 7106 -5682 5179 -9377 -4650 -3066 -2303 1547 -4785 -6370 8746 2938 -6204 -6126 286 -7457 6230 -7266 -754 -9445 -3543 6113 -3450 -9821 859 2326 9617 -8577 -6767 5844 2334 -492 4923 -2580 -9714 1883 8027 -3936 1100 1875 4541 4149 -8040 -5093 1939 5982 -4146 3270 -3996 -2481 5820 1276 9536 6862 -3322 7856 4430 -7666 6475 9686 4603 8529 7478 -4581 242 1612 639 -7080 -8365 -3952 -1950 -8820 -781 -96 1303 5460 2837 9671 -464 2712 6141 -6603 3663 -6970 -7903 -8848 -1388 9263 5228 -442 -7932 -493 -3291 1080 -6595 7965 5409 -6339 8944 -9227 8499 8816 2388 -2901 -3076 8238 7937 6923 -5538 6714 -2873 926 2614 -1146 7960 -2222 9831 185 6848 7939 -398 5241 -4377 1620 -494 -5249 6203 -8907 -7638 -1490 -823 -4845 4069 -5282 4485 -3189 -8369 7584 -2236 2534 5699 -1046 2345 7278 -2946 -2480 1898 -3378 -7413 -464 6335 5180 6820 8244 -4007 4729 -82 -4040 738 -955 -9995 -1558 8024 -5090 -1017 722 -8556 -5490 -7744 -8525 -3546 -4833 8660 5896 5208 -1153 5428 -9844 -9573 6868 -1897 -5103 -2548 -3283 -989 -9836 -4401 2915 9476 8115 7567 2290 6294 -2546 -4898 2220 9863 7619 6877 -4690 1264 3663 -2134 -2916 -1643 7681 -1913 -6614 -1294 -591 -3190 -5352 1352 5084 5257 8908 1994 -3323 -1446 2938 -8137 6779 8018 4682 9575 3789 6743 -5846 -8008 -1713 4330 -1675 -6729 -6669 9874 9768 -1103 -847 1010 -6113 -9026 2329 -8045 -6012 -4810 758 7348 -2845 7343 -7660 -837 -3610 -4143 8118 4329 13 6605 -6904 -6544 3464 -8812 1482 -2720 -8424 9139 9910 7176 -9043 -949 -6670 