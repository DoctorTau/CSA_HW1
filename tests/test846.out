-9762 -9348 -1114 -888 -5240 -6238 -5448 -5524 -3314 -7094 -3514 -8204 -3180 -6566 -9516 -2050 -1742 -630 -2426 -1752 -1400 -2660 -9420 -4800 -7734 -8398 -5988 -9250 -7286 -1296 -6154 -9382 -2240 -2858 -4544 -7552 -5882 -2908 -3604 -5610 -258 -6370 -2184 -8492 -1504 -5722 -5564 -7622 -8646 -9864 -202 -1520 -9058 -2966 -4982 -4318 -170 -4964 -2766 -4392 -5254 -7204 -5682 -4650 -3066 -6370 -6204 -6126 -7266 -754 -3450 -492 -2580 -9714 -3936 -8040 -4146 -3996 -3322 -7666 -7080 -3952 -1950 -8820 -96 -464 -6970 -8848 -1388 -442 -7932 -3076 -5538 -1146 -2222 -398 -494 -7638 -1490 -5282 -2236 -1046 -2946 -2480 -3378 -464 -82 -4040 -1558 -5090 -8556 -5490 -7744 -3546 -9844 -2548 -9836 -2546 -4898 -4690 -2134 -2916 -6614 -1294 -3190 -5352 -1446 -5846 -8008 -9026 -6012 -4810 -7660 -3610 -6904 -6544 -8812 -2720 -8424 -6670 